protocol/x/prices/keeper/market_test.go

```diff
@@ -57,6 +57,11 @@ func TestCreateMarket(t *testing.T) {
 
 	// Verify expected market event.
 	keepertest.AssertMarketCreateEventInIndexerBlock(t, keeper, ctx, marketParam)
+
+	// Verify market revenue share creation
+	revShareDetails, err := revShareKeeper.GetMarketMapperRevShareDetails(ctx, marketParam.Id)
+	require.NoError(t, err)
+	require.True(t, revShareDetails.ExpirationTs > 0)
```

Comment:
nit: Can you verify the value is some specific number rather than just > 0?

Reply:
The value will be set based on ctx.Blocktime which will be variable so I can't test for that.
Unless there's a way to get a deterministic time?

Reply:
Can't you get ctx.Blocktime and check against some variable that is blocktime + valid days?